README.md

```diff
@@ -97,6 +97,10 @@ git push
 # go online and request a pull
 ```
 
+Alternatively, you can also build and run the app in a pre-configured online workspace:
+
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/pcottle/learnGitBranching/blob/master/src/js/git/index.js)
```

Comment:
oh so the link is the file that opens, but it reads the yaml file automatically?

Reply:
Yes, after the `gitpod.io/#` prefix, you can use basically any GitHub URL (specific file, branch, pull request...) and it will try to do the right thing. It will also look for a `.gitpod.yml` config file at the root of the repository, to see if there is anything specific to do to set up that project.

I chose to pre-open `src/js/git/index.js` in the editor, but feel free to change it if you have a better idea.